### PR TITLE
Fix deployment rockspec

### PR DESCRIPTION
**Pull request by vadi2**
I think that's all I had to do to get this example working.

### CODE REVIEW

**Comment by leafo** [comment]
The development version of lapis was required?

**Comment by vadi2** [comment]
Well, mostly "lapis" on its own was not working - heroku was not building lapis and all of its dependencies. Specifying the rockspec explicitly worked. Perhaps you don't have to use the development rockspec.

File: app.rockspec [hunk 1]
```diff
@@ -1,5 +1,5 @@
 dependencies = {
-	"https://raw.github.com/leafo/heroku-openresty/master/heroku-openresty-dev-1.rockspec",
+	"https://raw.github.com/leafo/heroku-openresty/master/heroku-openresty-dev-2.rockspec",
```
Comment (mathieujobin) [inline app.rockspec:2]: i confirm, using `-2` worked for me, don't need the `lapis-dev-1` line below...

**Comment by daelvn** [comment]
I've just been having this problem with my own application. I tried forking this example repository and deploying it and the exact same error seems to be happening: Lapis is not building, only OpenResty. I tried changing it to dev-2 in my app to match the fix done here, but it still wouldn't work, so perhaps this could be retried?
This is the build log for the example:
```
-----> Lua app detected

-----> Copying lua to bin

-----> Installing packages

       Cloning into 'heroku-openresty'...

       Using https://raw.github.com/leafo/heroku-openresty/master/heroku-openresty-dev-2.rockspec... switching to 'build' mode

       		LIB_DIR=`cd /tmp/build_e0c0fb5165f78ddefa50a35d679c0d1c/packages/lib/luarocks/rocks/heroku-openresty/dev-2/../../../../; pwd`

       		BIN_DIR=`cd $LIB_DIR/../../bin; pwd`

       		STACK=${STACK:-cedar}

       		cp luajit/lib/libluajit-5.1.so.2.1.0 "$LIB_DIR"

       		(

       			cd "$LIB_DIR"

       			ln -s libluajit-5.1.so.2.1.0 libluajit-5.1.so.2

       			ln -s libluajit-5.1.so.2.1.0 libluajit-5.1.so

       		)

       		cp nginx/sbin/$STACK/nginx "$BIN_DIR"

       	

       No existing manifest. Attempting to rebuild...

       Updating manifest for /tmp/build_e0c0fb5165f78ddefa50a35d679c0d1c/packages/lib/luarocks/rocks

       Cloning into 'lua-date'...

       heroku-openresty dev-2 is now built and installed in /tmp/build_e0c0fb5165f78ddefa50a35d679c0d1c/packages (license: MIT)

       

       Using https://raw.github.com/leafo/lua-date/master/date-dev-1.rockspec... switching to 'build' mode

       Updating manifest for /tmp/build_e0c0fb5165f78ddefa50a35d679c0d1c/packages/lib/luarocks/rocks

       date dev-1 is now built and installed in /tmp/build_e0c0fb5165f78ddefa50a35d679c0d1c/packages (license: Public Domain)

       

-----> Discovering process types

       Procfile declares types -> web

-----> Compressing...

       Done: 6.7M

-----> Launching...

       Released v5

       https://lapis-ex.herokuapp.com/ deployed to Heroku
```
Mine is similar, except I don't have the date library and it *seems* to be detecting the rockspec, but not getting to install it. Strangely enough, specifying the rockspec location did not fix it either.

**Pull request closed without merging**